lib/AgentSDK.js

```diff
@@ -60,7 +60,14 @@ class SDK extends Events {
                     user: this.userId,
                     accountId: conf.accountId
                 }, (err, data) => {
-                    this.agentPID = data && data.pid;
+                    // TODO: Ugly hack, remove it later.
+                    // Currently there is a bug in the ac-users.
+                    // Transoformers adds convDetails.getMyRole() and ms.MessagingEvents.isMe()
+
+                    // Uncomment this line after fixing ac-users
+                    // this.agentId = data && data.pid;
+                    this.agentId = {error: "agentId not supported yet. Use convDetails.getMyRole() and ms.MessagingEvents.isMe()."};
```

Comment:
adding the function `convDetails.getMyRole()` means it is another breaking change in the next version.
If we actually decide to go with it we should add it to the documentation as well in the disclaimer part